2024-01-27 03:36:28,293: root: INFO: mltrader : Executing the initialize lifecycle method
2024-01-27 03:36:28,915: root: INFO: Current backtesting datetime 2023-12-15 08:30:00-05:00
2024-01-27 03:36:28,915: root: INFO: mltrader : Executing the before_market_opens lifecycle method
2024-01-27 03:36:28,916: root: INFO: Current backtesting datetime 2023-12-15 09:30:00-05:00
2024-01-27 03:36:28,916: root: INFO: mltrader : Executing the before_starting_trading lifecycle method
2024-01-27 03:36:28,917: root: INFO: [34mmltrader : Executing the on_trading_iteration lifecycle method at 2024-01-27 03:36:28[0m
2024-01-27 03:36:28,919: root: WARNING: quote is not implemented for YahooData, but USD was passed as the quote
2024-01-27 03:36:28,922: root: WARNING: quote is not implemented for YahooData, but USD was passed as the quote
2024-01-27 03:36:28,923: root: ERROR: Strategy.create_order() got an unexpected keyword argument 'stop_loss'
2024-01-27 03:36:28,923: root: ERROR: Traceback (most recent call last):
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 829, in run
    self._run_trading_session()
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 780, in _run_trading_session
    self._on_trading_iteration()
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 274, in func_output
    result = func_input(self, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 298, in func_output
    result = func_input(self, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 87, in func_output
    return func_input(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\strategies\strategy_executor.py", line 378, in _on_trading_iteration
    on_trading_iteration()
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\tools\decorators.py", line 62, in func_output
    frame, result = call_function_get_frame(func_input, *args, **kwargs)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\Ishan Roy Barman\AppData\Roaming\Python\Python311\site-packages\lumibot\tools\decorators.py", line 30, in call_function_get_frame
    result = func(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^
  File "c:\Users\Ishan Roy Barman\Desktop\algo\tradingbot.py", line 31, in on_trading_iteration
    order = self.create_order(
            ^^^^^^^^^^^^^^^^^^
TypeError: Strategy.create_order() got an unexpected keyword argument 'stop_loss'

2024-01-27 03:36:28,926: root: INFO: mltrader : Executing the on_bot_crash event method
2024-01-27 03:36:28,931: root: INFO: Backtesting finished
2024-01-27 03:36:28,932: root: WARNING: Cannot plot returns because the strategy returns are missing
2024-01-27 03:36:28,987: root: WARNING: Cannot create a tearsheet because the strategy returns are missing
2024-01-27 03:36:28,987: backtest_stats: INFO: Backtest took 0:00:00.694949 for a speed of 0.000
